mod/wikirate_source/set/type/source/source_preview.rb

```diff
@@ -94,7 +131,9 @@
               <span class="icon-bar"></span>
             </button>
             <div id="source-preview-tabs" class="navbar-brand" href="#">
-              #{web_link("/", :text=>raw( nest Card["*logo"], :view=>:core, :size=>:original ))}
+              #{web_link('/',
+                         text: raw(nest Card['*logo'],
```

Comment:
Add parentheses to nested method call `nest Card['*logo'],
                                        view: :core, size: :original`.